sdk/storage/azure-storage-file/tests/test_share_async.py

```diff
@@ -792,13 +789,13 @@ def test_list_directories_and_files_with_snapshot_async(self):
 
         # Act
         result = share_name.list_directories_and_files(results_per_page=2)
```

Comment:
Shouldn't this be `.by_page()`?

Reply:
Yes, if you're expected two, indeed should be a by_page. Look at the sync test, I guess I messed up my copy/paste

Reply:
fixed!